README.md

```diff
@@ -1 +1,75 @@
-# csaf-validator-service
\ No newline at end of file
+<div id="top"></div>
+
+<br />
+<div align="center">
+<h3 align="center">csaf-validator-service</h3>
+</div>
+
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+    </li>
+    <li>
+      <a href="#development">Development</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+        <li><a href="#run-server">Run server</a></li>
+      </ul>
+    </li>
+    <li><a href="#testing">Testing</a></li>
+  </ol>
+</details>
+
+## About The Project
+
+This is a service to validate documents against the CSAF standard. It uses the csaf-validator-lib under the hood which included as a git subtree module.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Documentation
+
+The documentation is available as a swagger resource provided by the service itself under `/docs`. So once the server is running, visit http://localhost:3000/docs in your browser.
```

Comment:
> http://localhost:3000/docs

Is that fix or definable/dependent by variables?

Reply:
This is definable with using configuration files in `config/`. I'll note this here.